sites/fast-website/src/app/css/style.css

```diff
@@ -187,20 +187,39 @@ fast-divider::after {
 }
 
 .section-badge {
-  color: var(--accent-fill-rest);
+  color: var(--accent-foreground-rest);
   font-weight: bold;
+  background: var(--neutral-fill-focus);
+  width: max-content;
+  align-self: center;
+  letter-spacing: 0.12em;
+  border-radius: calc(var(--corner-radius) * 1px);
+  padding: calc(var(--design-unit) * 1px) calc(var(--design-unit) * 4px);
```

Comment:
The badge supports a fill by default - I would expect us to use the API there to make this change. There are examples of how to create a map for the badge in the fixtures.

Reply:
@chrisdholt Thank you for the highlighting those attributes. They were very straightforward to assign, but the issue that I am having is that the fill and color are applied to the div inside of the template with the class badge and not to the fast-badge itself. So to apply the padding and the other stylings, I need to be able to assign them to that internal div with the badge class as well from the main css file, and this is proving quite challenging. In the past for buttons I have been able to use part to access internal elements within a template, but since this template does not include a part, is there another way to do it in this case?